egs2/TEMPLATE/st2/st2.sh

```diff
@@ -0,0 +1,2066 @@
+#!/usr/bin/env bash
```

Comment:
The script looks great!

One issue remains on my side is the support of multi-reference scenarios for ST evaluation (I double checked the support, but it seems not supported in preparation yet) Please refer  https://github.com/espnet/espnet/blob/master/egs2/TEMPLATE/st1/st.sh#L547-L553 for some details on how we process those

Reply:
For more information on the name:
https://github.com/espnet/espnet/blob/master/egs2/TEMPLATE/st1/st.sh#L1585C75-L1585C75